code/game/turfs/change_turf.dm

```diff
@@ -58,6 +58,13 @@ GLOBAL_LIST_INIT(blacklisted_automated_baseturfs, typecacheof(list(
 /turf/proc/ChangeTurf(path, list/new_baseturfs, flags)
 	if(!path)
 		return
+	if(path == /turf/baseturf_bottom)
+		path = SSmapping.level_trait(z, ZTRAIT_BASETURF) || /turf/open/space
+		if (!ispath(path))
```

Comment:
The trait should probably be converted to a path earlier than here

Reply:
I suppose it could go in SSmapping init when we load traits from the json but it seems like it'd be a little duplication heavy, if `text2path` is heavy we could store back to the traits list here